### ISSUE TITLE
Why default toleration in DS example k8s.io/docs/concepts/workloads/controllers/daemonset/

### ISSUE THREAD

**Issue by sam-sla** [kubernetes/website#19512]
**This is a Bug Report**

**Problem:**
The example "Create a DaemonSet" adds a toleration for the "NoSchedule" effect on the master node, I don't understand why. I don't think this is a good default to have in the example. 

**Proposed Solution:**
Remove the toleration from the spec:
```
    spec:
      tolerations:
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
```

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#create-a-daemonset

**Comment by sftim** [comment]
That seems like a valid concern.

/priority backlog
/good-first-issue
/language en

**Comment by sam-sla** [comment]
I see that for https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/ the example is more descriptive, maybe this is the way to go instead of removing the toleration. I can create a PR adding the same comment to the daemonset.yaml

**Comment by davivcgarcia** [comment]
@sam-sla is this fixed by PR #19530?

**Comment by sam-sla** [comment]
> 
> 
> @sam-sla is this fixed by PR #19530?

Yes @davivcgarcia a comment was added making it more clear 👍

**Comment by sftim** [comment]
/close

based on https://github.com/kubernetes/website/issues/19512#issuecomment-60793298
